test-app/ember-cli-build.js

```diff
@@ -4,20 +4,11 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function (defaults) {
   const app = new EmberApp(defaults, {
-    // Add options here
+    autoImport: {
+      watchDependencies: ['ember-engines-router-service'],
```

Comment:
@SergeAstapov why do we need this config?

Reply:
@villander this tells `ember-auto-import` to pick up changes in the addon.

This makes `ember s` and `ember test --server` commands pick up changes in the `ember-engines-router-service` addon without restarting the command.

This is not unique to `ember-engines-router-service`, it's just a config now aded by blueprint https://github.com/embroider-build/addon-blueprint/blob/main/files/test-app-overrides/ember-cli-build.js#L9 and that was just not present in this repo before.

Reply:
@SergeAstapov could you please enable it for non-production environments?

Reply:
@villander this flag only affects `ember test --server` and `ember server` commands, hence this makes no effect for `ember build` command.

Reply:
Thank you @SergeAstapov